bundles/org.openhab.binding.http/README.md

```diff
@@ -35,30 +37,30 @@ Authentication might fail if redirections are involved as headers are stripper p
-
 ## Channels
 
+The thing has two channels of type `request-date-time` which provide the timestamp of the last successful (`last-success`) and last failed (`last-failure`) request.
+
+Additionally, the thing can be extended with data channels.
 Each item type has its own channel-type.
 Depending on the channel-type, channels have different configuration options.
 All channel-types (except `image`) have `stateExtension`, `commandExtension`, `stateTransformation`, `commandTransformation` and `mode` parameters.
-The `image` channel-type supports `stateExtension`, `stateContent` and `escapedUrl` only.
-
-| parameter               | optional | default     | description |
-|-------------------------|----------|-------------|-------------|
-| `stateExtension`        | yes      |      -      | Appended to the `baseURL` for requesting states. |
-| `commandExtension`      | yes      |      -      | Appended to the `baseURL` for sending commands. If empty, same as `stateExtension`. |
-| `stateTransformation`   | yes      |      -      | One or more transformation applied to received values before updating channel. |
-| `commandTransformation` | yes      |      -      | One or more transformation applied to channel value before sending to a remote. |
-| `escapedUrl`            | yes      |      -      | This specifies whether the URL is already escaped. |
-| `stateContent`          | yes      |      -      | Content for state requests (if method is `PUT` or `POST`) |
+The `image` channel-type supports `stateExtension` only.
+
+| parameter               | optional | default     | description                                                                                               |
+|-------------------------|----------|-------------|-----------------------------------------------------------------------------------------------------------|
+| `stateExtension`        | yes      | -           | Appended to the `baseURL` for requesting states.                                                          |
+| `commandExtension`      | yes      | -           | Appended to the `baseURL` for sending commands. If empty, same as `stateExtension`.                       |
+| `stateTransformation  ` | yes      | -           | One or more transformation applied to received values before updating channel.                            |
+| `commandTransformation` | yes      | -           | One or more transformation applied to channel value before sending to a remote.                           |
+| `stateContent`          | yes      | -           | Content for state requests (if method is `PUT` or `POST`)                                                 |
 | `mode`                  | no       | `READWRITE` | Mode this channel is allowed to operate. `READONLY` means receive state, `WRITEONLY` means send commands. |
 
-Transformations need to be specified in the same format as
+Transformations need to be specified in the same format as 
```

Comment:
Did you mean to complete this sentence, since a space was added at the end?

Reply:
No. It should be deleted.